File: Beginner/Tanu and Head-bob (HEADBOB)/headbob.java
import java.util.*;



class Codechef {
	public static void main (String[] args) throws java.lang.Exception {
		Scanner scan = new Scanner(System.in);
		if(scan.hasNext()){
		    int t = scan.nextInt();
		    for(int c=0; c<t; c++){
		        int n = scan.nextInt();
		        String S = scan.next();
		        if(S.indexOf('I')!=-1)
		            System.out.println("INDIAN");
		        else if (S.indexOf('Y')!=-1)
		            System.out.println("NOT INDIAN");
		        else 
		            System.out.println("NOT SURE");
		    }
        }
        scan.close();
	}
}

File: Beginner/Tanu and Head-bob (HEADBOB)/headbob.cpp
#include <iostream>
using namespace std;
int main()
{
    string s;
    int t,i,l;
    cin>>t;
   while(t--)
   {cin>>l;
       cin>>s;
       
       for(i=0;i<l;i++)
       {
       if(s.at(i)=='I')
       {cout<<"INDIAN"<<endl;break;}
       if(s.at(i)=='Y')
       {cout<<"NOT INDIAN"<<endl;break;}
       
       }
       if(i>=l)
       cout<<"NOT SURE"<<endl;
       
   }
}